README.md

```diff
@@ -0,0 +1,23 @@
+![](https://github.com/sahabpardaz/nexus-tag-plugin/workflows/.github/workflows/test.yml/badge.svg)
+# Tag Plugin for Nexus Repository OSS
+
+This plugin provides tagging support to Nexus Repository OSS 3.x. Note that this feature is available on
+Nexus Repository Pro and it has some advantage above this plugin (better support, better integration, ...). So, it's
+intended to be used by whom can not buy Pro version.
+
+# Installation
+Download latest release and copy jar bundle to deploy directory of target nexus repository manager.
+For more information about installing plugins, see 
+"[Installing a custom Nexus 3 plugin](https://sonatype-nexus-community.github.io/nexus-development-guides/plugin-install.html)".
+
+# Usage
```

Comment:
Yeah I know, but I wanted to find a way to embed swagger doc here (with a link or sth else), but I couldn't find a neat way. But now I think this can be kept for another pull request, as I want to test this completely and then when we are sure about overall design, we document it as I think it's not going to be used by the world too soon.

Reply:
Done